docs/rancher.md

```diff
@@ -0,0 +1,37 @@
+# Koku Development Documentation
+
+## Debugging Celery Workers and Remote Attach with VSCode
+
+### Celery Worker Auto-Reload Issue
+
+One of the issues we've encountered in our development workflow is related to Celery workers not auto-reloading with Rancher. This can be inconvenient as it requires manual intervention to reload the worker when making changes.
+
+**Issue:** The Celery worker does not auto-reload with Rancher, leading to development challenges.
+
+**Solution:** We need to address this issue to streamline our development process.
```

Comment:
I would remove this whole section because we've fixed the auto-reload by tapping into Django auto-reload instead of relying on watchdog.